azure/resource-containers/pf-management-group/classes/pf-management-group-options.ps1

```diff
@@ -0,0 +1,17 @@
+Class PfManagementGroupOptions : PfOptions {
+    [string]$Name
+    [string]$ParentManagementGroupName
+    [bool]$PersistState=$true
+
+    PfManagementGroupOptions(){}
+    PfManagementGroupOptions([PfAzureContext]$PfAzureContext){
+        $this.SetName([PfAzureContext]$PfAzureContext)
+    }
+    [void]SetName([PfAzureContext]$PfAzureContext){
```

Comment:
SetName with arg of PfAzureContext feels little funky.   Seems like could just use 'default behavior' and not need to call this, or just pass the name you want.